juniper/src/executor/look_ahead.rs

```diff
@@ -544,7 +569,7 @@ query Hero {
 
     #[test]
     fn check_query_with_argument() {
-        let docs = ::parse_document_source(
+        let docs = parse_document_source(
```

Comment:
Why does this not need the turbofish and the other calls do?

Reply:
Because this is the only test that explicitly uses a `DefaultScalarValue` in it's expected results.
All other tests don't use scalar values, so we must tell rustc which scalar value type should be used.